crates/re_data_ui/src/entity_db.rs

```diff
@@ -54,12 +57,95 @@ impl crate::DataUi for re_entity_db::EntityDb {
                 ui.label(store_source.to_string());
                 ui.end_row();
 
-                // We are in the recordings menu, we know the kind
-                if false {
-                    re_ui.grid_left_hand_label(ui, "Kind");
-                    ui.label(store_kind.to_string());
-                    ui.end_row();
+                re_ui.grid_left_hand_label(ui, "Kind");
+                ui.label(store_kind.to_string());
+                ui.end_row();
+            }
+
+            {
+                re_ui.grid_left_hand_label(ui, "Size");
+                ui.label(re_format::format_bytes(self.total_size_bytes() as _))
+                    .on_hover_text(
+                        "Approximate size in RAM (decompressed).\n\
+                         If you hover an entity in the streams view (bottom panel) you can see the size of individual entities.");
+                ui.end_row();
+            }
+
+            if let Some(data_source) = &self.data_source {
+                re_ui.grid_left_hand_label(ui, "Data source");
+                data_source_button_ui(ctx, ui, data_source);
+                ui.end_row();
+            }
+        });
+
+        if ctx.store_context.is_active(self.store_id()) {
+            ui.add_space(8.0);
+            match self.store_kind() {
+                StoreKind::Recording => {
+                    ui.label("This is the active recording");
                 }
+                StoreKind::Blueprint => {
+                    ui.label("This is the active blueprint");
+                }
+            }
+        }
+
+        sibling_stores_ui(ctx, ui, self);
```

Comment:
small thing but I found showing the sibling stores on hover a bit much - should do a check on `verbosity` here imho and only show them on full